modules/qtwidgets/src/rawdatareaderdialogqt.cpp

```diff
@@ -198,25 +205,34 @@ RawDataReaderDialogQt::RawDataReaderDialogQt() {
     byteOffset_->setRange(0, std::numeric_limits<int>::max());
     byteOffset_->setValue(0);
     byteOffset_->setSuffix(" Byte");
+    byteOffsetLabel->setFocusProxy(byteOffset_);
     /*
     QLabel* timeStepOffsetLabel = new QLabel("Time step offset");
     timeStepOffset_ = new QSpinBox();
     timeStepOffset_->setRange(0, 4096);
     timeStepOffset_->setValue(0);
     timeStepOffset_->setSuffix(" Byte");
     */
-    QLabel* endianessLabel = new QLabel("Endianess");
-    endianess_ = new QComboBox();
-    endianess_->addItem("Little Endian");
-    endianess_->addItem("Big Endian");
+    byteOrder_ = new QComboBox();
+    byteOrder_->addItem("Little Endian", QVariant::fromValue(iff::ByteOrder::LittleEndian));
+    byteOrder_->addItem("Big Endian", QVariant::fromValue(iff::ByteOrder::BigEndian));
+    QLabel* endianessLabel = new QLabel("Byte order");
+    endianessLabel->setFocusProxy(byteOrder_);
     readOptionsLayout->addWidget(byteOffsetLabel, 0, 0);
     readOptionsLayout->addWidget(byteOffset_, 0, 1);
     /*
     readOptionsLayout->addWidget(timeStepOffsetLabel, 1, 0);
     readOptionsLayout->addWidget(timeStepOffset_, 1, 1);
     */
-    readOptionsLayout->addWidget(endianessLabel, 2, 0);
-    readOptionsLayout->addWidget(endianess_, 2, 1);
+    readOptionsLayout->addWidget(endianessLabel, 1, 0);
+    readOptionsLayout->addWidget(byteOrder_, 1, 1);
+
+    useCompression_ = new QCheckBox{};
+    QLabel* useCompressionLabel = new QLabel("Compressed data (bzip, gz, lzma)");
```

Comment:
No need to be so specific with the compression algorithm. Suggest to simply specify zip. 
```suggestion
    QLabel* useCompressionLabel = new QLabel("Compress (zip)");
```

Reply:
why you want to know that formats that can be decompressed. And zip is not even really a valid one.

Reply:
Yeah, that was my thinking too. I put zip there as I thought there may have been a special reason to specify the compression method